src/Data/Ord.purs

```diff
@@ -214,10 +214,12 @@ between low hi x
 abs :: forall a. Ord a => Ring a => a -> a
 abs x = if x >= zero then x else negate x
 
--- | The sign function; always evaluates to either `one` or `negate one`. For
--- | any `x`, we should have `signum x * abs x == x`.
+-- | The sign function; returns `one` if the argument is positive, `negate one` if the argument is negative, or `zero` if the argument is `zero`. For any `x`, we should have `signum x * abs x == x`.
```

Comment:
Since `signum` now returns `x` instead of `zero`, maybe the docs should mention this?

```suggestion
-- | The sign function; returns `one` if the argument is positive, `negate one` if the argument is negative, or `x` if the argument is `zero`. For any `x`, we should have `signum x * abs x == x`.
```

Also, maybe the comment should be broken into multiple lines? This is both more consistent with the rest of the file and easier to read.

Reply:
While this is more accurate, I think it’s also likely to seem a weird way of wording it, especially if people aren’t thinking of Number or aren’t aware of how floats have signed zeroes. Also it might not be clear to people who are reading HTML docs what `x` is here.  Maybe it’d be better to leave this sentence as is and add another one at the end saying that for floating point numbers with signed zeroes, when called with a zero, this function returns the argument in order to preserve the sign?

Reply:
How's the new rendering